README.md

```diff
@@ -10,13 +10,13 @@ This package enriches your Fivetran data by doing the following:
 
 ## Models
 
-This package contains staging models, designed to work simultaneously with our [Pardot modeling package](https://github.com/fivetran/dbt_pardot). The staging models:
+This package contains staging models, designed to work simultaneously with our [Pardot modeling package](https://github.com/fivetran/dbt_pardot). The staging models are designed to:
 
 * Remove any rows that are soft-deleted
 * Name columns consistently across all packages:
-* Boolean fields are prefixed with `is_` or `has_`
-* Timestamps are appended with `_timestamp`
-* ID primary keys are prefixed with the name of the table. For example, the prospect table's ID column is renamed prospect.
+  * Boolean fields are prefixed with `is_` or `has_`
+  * Timestamps are appended with `_timestamp`
+  * ID primary keys are prefixed with the name of the table. For example, the prospect table's ID column is renamed `prospect_id`.
```

Comment:
Please verify - based on the content, I assumed that “prospect” was meant to be a prefix.

Reply:
this is correct, great catch!